docs/source/changelog.rst

```diff
@@ -34,9 +37,14 @@ Bug fixes
   buildpack in :pr:`633` by :user:`betatim`.
 - Update to version 5.7.6 of the `notebook` package used in all environments
   in :pr:`628` by :user:`betatim`.
+- Update to version 5.7.8 of the `notebook` package and version 2.0.12 of
+  `nteract-on-jupyter` in :pr:`650` by :user:`betatim`.
 - Switch to newer version of jupyter-server-proxy to fix websocket handling
   in :pr:`646` by :user:`betatim`.
-- Update to pip v19.0.3 in :pr:`647` by :user:`betatim`.
+- Update to pip version 19.0.3 in :pr:`647` by :user:`betatim`.
+- Ensure ENTRYPOINT is an absolute path in :pr:`657` by :user:`yuvipanda`.
+- Fix handling of `--build-memory-limit` values without a postfix in :pr:`652`
+  by :user:`betatim`.
 
```

Comment:
```suggestion
- Create a page naming and describing the "Reproducible Execution
  Environment Specification" (the specification used by repo2docker)
  in :pr:`656` by :user:`choldgraf`.

```

Reply:
shall we add this in and then release?

Reply:
Ah, this references the issue and not the PR. Let's add this to the change log in #662 itself. Should also be in the new features section, not "bug fix".

Reply:
-> I removed it again after pressing commit.

Reply:
Me again ... after having just merged #662 I added your sentence back :D.